### DISCUSSION TITLE
Can I "import" a sphinx generated site into the wiki?

### DISCUSSION THREAD

**Discussion by turekg** [requarks/wiki#4822] [Help / Questions] [answered]
We've got a sphinx generated web site  which we'd like to move into the wiki without making any major changes. In particular I am thinking of the site's sidebar which would have to replace the default wiki.js one. Is this even possible? 
Thanx!

**Comment by chintanp** [comment]
@turekg were you able to resolve this?

**Reply by turekg** [reply, marked as answer]
I got no replies to my question so what I had to do is import the pages and rework all the links.
